classes/external/change_kanban_content.php

```diff
@@ -1109,4 +1109,62 @@ public static function default_returns(): external_single_structure {
                 ]
             );
     }
+
+    /**
+     * Duplicate a card on the board.
+     *
+     * @param int $cmid
```

Comment:
`* @param int $cmid The course module id of the kanban board.
* @param int $boardid The id of the kanban board.`